exp-player/addon/components/exp-player.js

```diff
@@ -11,6 +11,21 @@ export default Ember.Component.extend(FullScreen, {
     session: null,
     frames: null,
 
+    store: Ember.inject.service(),
+    getCurrentUser: Ember.inject.service(),
+    pastSessions: Ember.computed('experiment', function() {
+        return this.get('getCurrentUser').load().then(([, profile]) => {
+            var profileId = profile.get('profileId');
+            var experimentId = this.get('experiment.id');
+            return this.get('store').queryRecord(this.get('experiment.sessionCollectionId'), {
+                q: [
+                    `experimentId:${experimentId}`,
+                    `profileId:${profileId}`
```

Comment:
It's worth noting that this returns a promise, not a value. Usually that should be ok, but (especially since this is tied to other properties that recompute when it changes), it's possible you may want to wait until the promise is resolved to return a value.